lib/vizkit/vizkit_items.rb

```diff
@@ -288,8 +288,13 @@ def update(data = nil)
                     end
                 else
                     @typelib_val = data
-                    rb_sample = @typelib_val.to_ruby
-                    @direct_type = DIRECTLY_DISPLAYED_RUBY_TYPES.any? { |rt| rt === rb_sample }
+                    if @typelib_val.class.convertion_to_ruby
```

Comment:
Needs conversion does not do the job because it excludes primitive types
But the check DIRECTLY_DISPLAYED_RUBY_TYPES does not include typelib primitives.
This is the easier way to check if this is a simply type.